templates/tv/index.html.twig

```diff
@@ -0,0 +1,229 @@
+                            <div class="input-group-prepend">
+                                <!-- Layout API, functionality initialized in Template._uiApiLayout() -->
+                                <button type="button" class="btn btn-danger" data-toggle="layout" data-action="header_search_off">
+                                    <i class="fa fa-fw fa-times-circle"></i>
+                                </button>
+                            </div>
+                            <input type="text" class="form-control" placeholder="Search or hit ESC.." id="page-header-search-input" name="page-header-search-input">
+                        </div>
+                    </form>
+                </div>
+            </div>
+            <!-- END Header Search -->
+
+            <!-- Header Loader -->
+            <!-- Please check out the Loaders page under Components category to see examples of showing/hiding it -->
+            <div id="page-header-loader" class="overlay-header bg-white">
+                <div class="content-header">
+                    <div class="w-100 text-center">
+                        <i class="fa fa-fw fa-circle-notch fa-spin"></i>
+                    </div>
+                </div>
+            </div>
+            <!-- END Header Loader -->
+        </header>
+        <!-- END Header -->
+
+        <!-- Main Container -->
+        <main id="main-container">
+
+            <!-- Hero -->
+            <div class="bg-body-light">
+                <div class="content content-full">
+                    <div class="d-flex flex-column flex-sm-row justify-content-sm-between align-items-sm-center">
+                        <h1 class="flex-sm-fill h3 my-2">
+                            List of TV Serials <small class="d-block d-sm-inline-block mt-2 mt-sm-0 font-size-base font-w400 text-muted">List of all recent TV serials</small>
+                        </h1>
+                        <nav class="flex-sm-00-auto ml-sm-3" aria-label="breadcrumb">
+                            <ol class="breadcrumb breadcrumb-alt">
+                                <li class="breadcrumb-item" aria-current="page">
+                                    <a class="link-fx" href="">Home</a>
+                                </li>
+                                <li class="breadcrumb-item">TV Serials</li>
+                            </ol>
+                        </nav>
+                    </div>
+                </div>
+            </div>
+            <!-- END Hero -->
+
+            <!-- Page Content -->
+            <div class="content">
+
+                <div class="row">
+                    {% for serial in serials %}
+                        <div class="col-md-4">
+                            <div class="block">
+                                <div class="block-header">
+                                    <h3 class="block-title">{{ serial.name }} </h3> <small>Original Language : {{ serial.originalLanguage }}</small>
```

Comment:
I think we could make something better with the display here. 
Maybe fix the max length and width, or something like that

Reply:
Not the final design. It will be reworked later.